contents/docs/notebooks/index.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: Notebooks
+sidebar: Docs
+showTitle: true
+availability:
+    free: full
+    selfServe: full
+    enterprise: full
+---
+
+> Notebooks are currently in **public beta**. This means it's not yet a perfect experience, but we'd love to know your thoughts. Please [share your feedback](http://app.posthog.com/home#supportModal) and [follow our roadmap](https://github.com/PostHog/posthog/issues/15680)!
+
+Notebooks enable you to combine the tools of PostHog into a single space for analysis. Rather than jumping between dashboards, insights, replays, experiment results, and more, your chain of analysis exists on a single page.
+
+It provides a flexible interface for organizing ad-hoc analysis, bug investigations, internal documentation, and feature releases by yourself or with your time.
+
+You can get started with notebooks by going to [the notebook tab](https://app.posthog.com/notebooks) in your PostHog instance and clicking "New notebook."
+
+## Accessing the notebook popover
+
+Critical to combining the functionality of PostHog is being able to access notebooks from anywhere, like a scratchpad for analysis which you can organize and clean up later.
+
+You can access a notebook by clicking the journal icon in the navbar (next to the notification bell) or by clicking and dragging a link anywhere in PostHog.
+
+![Popover icon](../../images/docs/notebooks/popover.png)
+
+To switch the notebook that opens in the popover, click the title button in the top left of the popover 
+
+![Switch popover notebook](../../images/docs/notebooks/switch.png)
+
+## Adding content to notebooks
+
+To start adding content to notebooks, start typing. There are three main ways to add content to notebooks:
+
+1. **Slash commands.** Within a notebook, type `/` and a menu will pop up with options for you. For example, `/trend` creates a trend insight.
+
+2. **Click the `+` icon on an empty line**. This opens the same menu as slash commands.
+
+3. **Dragging to popover.** From the other sections in PostHog, click and hold a link and a popover notebook shows up on the right side you can drag into. 
```

Comment:
I feel like these would for sure benefit from gifs / screenshots.

Reply:
Added one for each.